### PR TITLE
Add a bunch more strings

### PR DESCRIPTION
**Pull request by jimrandomh**
Boolean keywords. Numeric constants in funny formats. More kinds of XSS. Malformed URIs. Very longs strings to overflow buffers. The empty string.

### CODE REVIEW

File: blns.json [hunk 1]
```diff
@@ -91,10 +102,62 @@
...
...nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn",
```
Comment (davefp) [inline blns.json]: The README asks that you not add excessively long strings: https://github.com/minimaxir/big-list-of-naughty-strings#contributions

**Comment by minimaxir** [comment]
Please remove the 1000+ character strings and I'll merge.

**Comment by jimrandomh** [comment]
Not having any long strings seems like a mistake; that is something that generally needs testing. What if the long strings were machine-generated from the txt_to_json.py, rather than inline in blns.txt?

(The file is going to be hard to view no matter what. In fact, the first text editor I tried crashed on some of the unicode.)

**Comment by minimaxir** [comment]
The intent of the list is to hit edge cases. Long text in itself is not an edge case.  (For smaller forms, like email and names, long text may be more of an issue; however, 100+ characters is more than enough to hit validation issues there. )

**Comment by jomo** [comment]
> The first text editor I tried crashed on some of the unicode

Apparently the author is not testing against naughty strings :)

File: blns.json (hunk 1 above)
Comment (aesopwolf) [inline blns.json]: @davefp I don't want to defend whether long strings should be included or not, but here's one such example of a long string being a "naughty" one https://klikki.fi/adv/wordpress2.html

**Pull request closed without merging**